docs/source/app_developers_guide/adding_a_node.rst

```diff
@@ -0,0 +1,432 @@
+   the example configuration file, as in the following command:
+
+   .. code-block:: console
+
+      user@client-2$ sudo cp /sawtooth-core/packaging/validator.toml.example /etc/sawtooth/validator.toml
+
+#. Edit ``/etc/sawtooth/validator.toml`` to change the following settings:
+
+   a. Set the ``bind`` directive to the network and component endpoints for this
+      node. For this application development environment, use the default
+      values.
+
+      .. code-block:: none
+
+         bind = [
+           "network:tcp://127.0.0.1:8800",
+           "component:tcp://127.0.0.1:4004"
+         ]
+
+   #. Set ``endpoint`` to the publicly addressable IP and port of the
+      validator-to-validator connection. By default, ``endpoint`` is the same as
+      the bind ``network`` setting. It must be changed if the validator is
+      hosted behind a firewall. In that case, the firewall must map (redirect)
+      the endpoint IP:port to the bind network IP:Port.
+
+      .. code-block:: none
+
+         endpoint = "tcp://external-ip-2:8800"
+
+      The validator will share this endpoint IP:port with other validators when
+      when it is peering with them.
+
+   #. Select static peering by setting ``peers`` to the advertised network
+      endpoint of the first validator.
+
+      .. code-block:: none
+
+         peers = ["tcp://external-ip-1:8800"]
+
+      .. note::
+
+         If you are adding several nodes, specify the nodes in a comma-separated
+         list, as in this example:
+
+         .. code-block:: none
+
+            peers = ["tcp://external-ip-1:8800","tcp://external-ip-3:8800",...,"tcp://external-ip-5:8800"]
+
+   #. Ensure that the ``seeds`` directive is commented out.
+
+      .. note::
+
+         Static peering (the default) is appropriate for a Sawtooth network with
+         a small number of nodes (less than six or seven). Dynamic peering is
+         preferred if you plan to have many nodes on the network.
+
+         For more information on static and dynamic peering, see
+         :doc:`../sysadmin_guide/configuring_sawtooth/validator_configuration_file`
+         in the System Administrator's Guide and
+         :doc:`../cli/sawtooth-validator` in the CLI Command Reference.
+
+   #. Ensure that all other settings in this file match the settings for the
+      first validator node. These settings should be commented out unless you
+      made changes on the first node.
+
+Repeat this step to add more nodes to the Sawtooth network, if desired.
+
+Step 3: Restart the Sawtooth Network
+------------------------------------
+
+After making the configuration changes, restart all validator nodes in the
+Sawtooth network.
+
+#. Shut down the first validator node as described in
+   :ref:`stop-sawtooth-ubuntu-label`.
+
+#. Shut down the second validator node as described in
+   :ref:`stop-sawtooth-ubuntu-label`.
+
+#. Start all Sawtooth components on the first validator node, as shown in this
```

Comment:
I think, for the initial validator, you only need to restart the validator process for the changes listed above to take effect.

Reply:
Why wouldn't you start the REST API and transaction processors on the first validator node?  (The first command in this step shuts them all down.)